### ISSUE TITLE
Hyperparameters used for each dataset?

### ISSUE THREAD

**Issue by evantkchong** [dangweili/pedestrian-attribute-recognition-pytorch#36]
Hello there! Thank you for sharing this project on Github.

My team is having trouble achieving good performance for RAPv2 and PA100k. We have tried using a range of hyperparameters using what is in `./script/experiment/train.sh` as a baseline but we cannot achieve mean accuracy greater than 0.77 for the label based evaluation.

Do you mind sharing what hyperparameters were used for each dataset? We would greatly appreciate it. Thank you!
